cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc

```diff
@@ -127,22 +155,66 @@ ON [EACH] type(r)
 
 Index provider can be specified using the `OPTIONS` clause.
 
+| [source, cypher, role=noplay]
+----
+CREATE TEXT INDEX [index_name] [IF NOT EXISTS]
+FOR (n:LabelName)
+ON (n.propertyName)
+[OPTIONS "{" option: value[, ...] "}"]
+----
+| Create a text index on nodes.
+
+Index provider can be specified using the `OPTIONS` clause.
+
+| [source, cypher, role=noplay]
+----
+CREATE TEXT INDEX [index_name] [IF NOT EXISTS]
+FOR ()-"["r:TYPE_NAME"]"-()
+ON (r.propertyName)
+[OPTIONS "{" option: value[, ...] "}"]
+----
+| Create a text index on relationships.
+
+Index provider can be specified using the `OPTIONS` clause.
+
+| [source, cypher, role=noplay]
+----
+CREATE POINT INDEX [index_name] [IF NOT EXISTS]
+FOR (n:LabelName)
+ON (n.propertyName)
+[OPTIONS "{" option: value[, ...] "}"]
+----
+| Create a point index on nodes.
```

Comment:
Should we mention somewhere that point index only handles properties with point values?

Reply:
Sounds like a good idea, see my answer on the text index PR.